27/enClase/autos.ts

```diff
@@ -0,0 +1,61 @@
+abstract class Auto {
+    velocidad: number;
+    estaPrendido: boolean;
+    public constructor() {
+        this.estaPrendido = false;
+        this.velocidad = 0;
+    }
+    abstract acelerar(): void;
+    abstract frenar();
+    private prenderApagar() {
+        if (this.estaPrendido == true) {
```

Comment:
Otra forma de usar esta condicion es simplemente decir if (this.estaPrendido) 
O con un ternario 
private prenderApagar() {
this.estaPrendido =   this.estaPrendido ? false : true;